src/index.html

```diff
@@ -8,6 +8,76 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Stars</h1>
+    <div class="stars stars--0">
+      <div class="stars__star"></div>
+
+      <div class="stars__star"></div>
+
+      <div class="stars__star"></div>
+
+      <div class="stars__star"></div>
+
+      <div class="stars__star"></div>
+    </div>
+    
+    <div class="stars stars--1">
+      <div class="stars__star stars__active"></div>
```

Comment:
```suggestion
      <div class="stars__star"></div>
```
Remove it everywhere use nth-child instead

Reply:
Done, but is it okay if I have written it in many selectors? Like in the "Update №2" or there is a better and easier way to write with 'nth-child'?

Reply:
it's better to do it with 'nth-child' as you do :+1:

Reply:
Thanks)